CoffeeChat/Controller/MatchViewController.swift

```diff
@@ -28,29 +33,73 @@ class MatchViewController: UIViewController {
         MatchSummary(title: "He is also part of...", detail: "EzraBox")
     ]
 
+    init(hasReachedOut: Bool) {
+        self.hasReachedOut = hasReachedOut
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.navigationBar.isHidden = true
+    }
+
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupViews()
+        setupConstraints()
+    }
+
+    private func setupViews() {
         view.backgroundColor = .backgroundLightGreen
 
         // TODO: Remove after connecting to backend. These are temp values.
         let firstName = "Ezra"
         let lastName = "Cornell"
         let major = "Government"
         let year = 2020
+        let pronouns = "He/Him"
         let hometown = "Ithaca, NY"
+        let user = User(clubs: [], firstName: firstName, googleID: "", graduationYear: "2020", hometown: hometown,
+                        interests: [], lastName: lastName, major: major, matches: [], netID: "", profilePictureURL: "",
+                        pronouns: "pronouns", facebook: "https://www.facebook.com", instagram: "https://www.instagram.com")
+
+        if !hasReachedOut {
+            reachOutButton = UIButton()
```

Comment:
I think this might be an antipattern but I'm not sure. Alternatively, you could say `let reachOutButton = UIButton()`, remove the `if let`, and say `self.reachOutButton = reachOutButton` right before/after you `addSubview`.